llvm/docs/AMDGPUUsage.rst

```diff
@@ -576,6 +576,12 @@ Generic processor code objects are versioned. See :ref:`amdgpu-generic-processor
                                                                                                   - ``v_dot2_f32_f16``
 
 
+     ``gfx9-4-generic``   ``amdgcn``     - ``gfx940``      - xnack            - Absolute flat   FP8 and BF8 instructions,
```

Comment:
What about sramecc?

Reply:
It is supported.

Reply:
so should be added to target features supported column?